main/base-removed-dbg/template.py

```diff
@@ -145,6 +145,7 @@
     "qalc",
     "rfkill",
     "sdl",
+    "tcllib",
```

Comment:
doesn't this need a relbump

Reply:
that part just shouldn't be there because that's not what the package is for